includes/controls/settings-footer.php

```diff
@@ -0,0 +1,535 @@
+<?php
+namespace Elementor\Core\Kits\Documents\Tabs;
+
+use Elementor\Plugin;
+use Elementor\Controls_Manager;
+use Elementor\Core\Responsive\Responsive;
+
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+// Allow active/inactive via the Experiments
+if ( ! hello_header_footer_experiment_active() ) return;
+
+class Hello_Settings_Footer extends Tab_Base {
+
+	public function get_id() {
+		return 'hello-settings-footer';
+	}
+
+	public function get_title() {
+		return __( 'Footer', 'hello-elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-footer';
+	}
+
+	public function get_help_url() {
+		return '';
+	}
+
+	public function get_group() {
+		return 'theme-style';
+	}
+
+	protected function register_tab_controls() {
+		$this->start_controls_section(
+			'hello_footer_section',
+			[
+				'tab' => 'hello-settings-footer',
+				'label' => __( 'Footer', 'hello-elementor' ),
+			]
+		);
+
+		$this->add_control(
+			'hello_footer_logo_display',
+			[
+				'type' => Controls_Manager::SWITCHER,
+				'label' => __( 'Site Logo', 'hello-elementor' ),
+				'default' => ( false === get_option( 'hello_footer_type_default' ) ? 'yes' : false ),
```

Comment:
?

Reply:
I've given you this comment before, didn't notice it wasn't dealt with:

`get_option( 'hello_footer_type_default' )` is called multiple times here. Call it once at the top and assign it to a variable, then use that variable everywhere you need within this method.

Reply:
Rename the option, assign it to a variable. Check this condition and the option setting to make sure the correct value is applied.

Reply:
We've assigned this to a variable now

Reply:
Even better, we've removed this behavior entirely, now we only show the Footer elements when the experiment is active.

For users who update from an older version, the experiment defaults as `inactive`.

This works well for now, but we'll need to think about when we change this from an experiment to a full feature, in which case we may have to bring this logic back in some way.